blaze/server/server.py

```diff
@@ -62,6 +67,17 @@ class RC(object):
 _no_default = object()  # sentinel
 
 
+def _logging(func):
+    @wraps(func)
+    def _logger(*args, **kwargs):
+        logger = flask.current_app.logger
+        logger.debug("Calling %s" % func.__name__)
+        ret = func(*args, **kwargs)
+        logger.debug("Leaving %s" % func.__name__)
```

Comment:
Would it be useful to log "Leaving" in the exception case, i.e. should this be in a `finally`?